src/contexts/Api/defaults.ts

```diff
@@ -9,6 +9,26 @@ import {
   APIContextInterface,
   ConnectionStatus,
 } from 'contexts/Api/types';
+import { NetworkName } from 'types';
+
+const isNetworkName = (value: unknown): value is NetworkName =>
+  Object.values<unknown>(NetworkName).includes(value);
+
+const defaultNetworkName =
+  process.env.NODE_ENV === 'production' &&
+  process.env.REACT_APP_DISABLE_MAINNET !== '1'
+    ? NetworkName.AlephZero
+    : NetworkName.AlephZeroTestnet;
+
+const cachedNetworkName = localStorage.getItem('network');
+
+if (!isNetworkName(cachedNetworkName)) {
+  localStorage.setItem('network', defaultNetworkName);
+}
```

Comment:
This actually breaks the fix - the fact that the name exists, doesn't mean we have a config for it. We need to check if the network is set in networks[]. 
Right now on mainnet, you could have it set local storage "alephzerotestnet", which exists in the NetworkNames struct, but it is not configured in networks[]

Reply:
Great catch - fixing by chaking `NETWORKS` instead of `NetworkName`.

Reply:
Fixed.